Source/WebInspectorUI/UserInterface/Views/ColorPicker.css

```diff
@@ -26,7 +26,8 @@
 .color-picker {
     position: relative;
     width: var(--color-picker-width);
-    height: 236px;
+    /* needs workig it results in an extra scroll bar within the parent div called 'container' within popover I believe*/
+    max-height: 350px;
```

Comment:
im not sure this is the exact right value, as i'm still seeing a scrollbar

frankly, do we actually even need this?  having this here makes me even more strongly feel that we should get rid of `WI.ColorSwatchDisplay` entirely since it means that if the styles of that change, then this would also have to update in order to accommodate that, and i'd much rather not have to keep track of two different files at once